tech-far-hub/gatsby-node.ts

```diff
@@ -47,14 +48,17 @@ interface IPageContext {
   id: string;
   breadCrumbs: IBreadcrumb[];
   pathParts: string[];
+  parentPath: string;
+  parentPathRegex: string;
+  isIndex: boolean;
 }
 
 export const createPages: GatsbyNode["createPages"] = async ({ graphql, actions, reporter }) => {
   const { createPage } = actions;
 
   const result: IGraphQLTemplateNodeResult = await graphql(`
     query CreatePages {
-      allMdx {
+      allMdx(filter: { internal: { contentFilePath: { glob: "!*components" } } }) {
```

Comment:
does this mean All MDX without components?

Reply:
It's a path filter --- so it's basically "every file in the content directories EXCEPT components and its children," since components are just little chunks of reused content that shouldn't get pages.